packages/react/src/components/field-container/utils.ts

```diff
@@ -0,0 +1,59 @@
+import { ReactNode } from 'react';
+import { devConsole } from '../../utils/dev-console';
+import { joinStrings } from '../../utils/string';
+import { FieldSlotIds } from './types';
+
+export function getSlotId(prop: ReactNode, fieldId: string, propName: string): string | undefined {
+    return prop ? `${fieldId}_${propName}` : undefined;
+}
+
+export function getSlotIds(
+    fieldId: string,
+    label?: ReactNode,
+    hint?: ReactNode,
+    invalid?: ReactNode,
```

Comment:
C'est un peu louche `ReactNode`. Tu pourrais ajouter un type `type Primitive = string | number | boolean | undefined | null;`
```suggestion
    label?: Primitive,
    hint?: Primitive,
    invalid?: Primitive,
```
Ou sinon peut-être que ça devrait être des booléens et que ce soit l'endroit où on call `getSlotIds` qui détermine si c'est true ou false?

Reply:
J'aime bien l'idée des boolean!